molgenis-core-ui/src/main/resources/js/jquery.molgenis.table.js

```diff
@@ -407,16 +406,10 @@
                 }
 				break;
             case 'BOOL':
-            	// FIXME refactor formatTableCellValue to accept attribute instead of attribute field type and move nillable boolean code to this function
```

Comment:
This suggestion, to send the entire attribute to the formatTableCellValue instead, sounds a bit nicer to me. Did you consider it?